targets/standard.props

```diff
@@ -2,9 +2,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <ProjectDirectory Condition="'$(PathToRepo)' != ''">$(PathToRepo)</ProjectDirectory>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration)</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) --</BuildCommand>
```

Comment:
what is the "--" for here?

Reply:
This tells run.exe to pass all remaining args to msbuild directly so that repo API args can be passed. (This PR doesn't change the repos to  implement the "core" repo API framework.)